packages/client/src/Spacetime.ts

```diff
@@ -35,12 +37,13 @@ export class Spacetime {
 
   collection<T=any> (id: string): Collection<T> {
     if (this.collections[id]) return this.collections[id]
```

Comment:
I think this should now be moved down a line and changed to `if (this.collections[path])`